Validation/HGCalValidation/plugins/HGCalTriggerValidator.cc

```diff
@@ -194,15 +203,16 @@ void HGCalTriggerValidator::bookHistograms(DQMStore::IBooker &iBooker,
 }
 
 void HGCalTriggerValidator::dqmAnalyze(edm::Event const &iEvent,
-                                       edm::EventSetup const &iSetup,
+                                       const edm::EventSetup &iSetup,
```

Comment:
This is semantically the same. There is no reason to change since the convention `Type const &` is used throughout this file and not `const Type &`